pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -148,6 +148,12 @@ type AlertmanagerSpec struct {
 	// and the actual ExternalURL is still true, but the server serves requests
 	// under a different route prefix. For example for use with `kubectl proxy`.
 	RoutePrefix string `json:"routePrefix,omitempty"`
+	// Maximum number of GET requests processed concurrently.
+	// +optional
+	GetConcurrency *uint32 `json:"getConcurrency,omitempty"`
+	// Timeout for HTTP requests.
```

Comment:
since it is unit32, I would expect k8s to reject negatives. Default being zero depends completely on the Alertmanager behaviour, which *might* change from version to version. I think it we could instead say, that this follows the same behaviour as the web.concurrency option of alertmanager when set to 0. 
WDTY?

Reply:
I agree with @sthaha.

Reply:
Yup agree, I'll try to keep that in mind for follow-up reviews! Thanks for making it clear 🙏